dk.tangramgames.mrrescue.json

```diff
@@ -0,0 +1,93 @@
+{
+  "app-id": "dk.tangramgames.mrrescue",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "mrrescue",
+  "finish-args": [
+    "--device=all",
+    "--socket=pulseaudio",
+    "--socket=x11"
+  ],
+  "modules": [
+    {
+      "name": "PhysicsFS",
+      "buildsystem": "cmake",
```

Comment:
`cmake-ninja`?

Reply:
That's a good idea, it should be faster and not break anything.